pkg/nimble/statconn/include/nimble_statconn.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    pkg_nimble_statconn Statconn
+ * @ingroup     pkg_nimble
+ * @brief       Static connection manager for NimBLE netif that keeps opens
+ *              connections on demand and takes care of keeping them open.
+ *
+ * @experimental
+ *
+ * # WARNING
+ * This module is highly experimental! Expect bugs, instabilities and sudden API
+ * changes :-)
+ *
+ *
+ * # About
+ * Statconn is the implementation of a static connection manager for the NimBLE
+ * netif module. It initiates new connections when told and additionally takes
+ * care of reopening connections in case of connection loss.
+ *
+ *
+ * # Usage
+ * A node can either be a master, the one initiating the connection, or a
+ * slave, the one advertising its presence, w.r.t. to BLE connections. To open
+ * a connection between two nodes, call nimble_statconn_add_slave() with the
+ * BLE address of the slave node on the designated master node and call
+ * nimble_statconn_add_master() with the address of the master node on the
+ * designated slave node. From that point on, statconn will take care of
+ * opening and maintaining the connection between these two nodes.
+ *
+ *
+ * # Configuration
+ * This module is configured completely statically. All relevant configuration
+ * options are set at compile time using a couple of macros.
+ *
+ * @{
+ *
+ * @file
+ * @brief       Simple static connection manager for NimBLE netif
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ */
+
+#ifndef NIMBLE_STATCONN_H
+#define NIMBLE_STATCONN_H
+
+#include <stdint.h>
+
+#include "nimble_netif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Advertising interval used when in advertising mode [in ms]
+ */
+#ifndef NIMBLE_STATCONN_ADV_ITVL
+#define NIMBLE_STATCONN_ADV_ITVL            (90U)
```

Comment:
Here and below: it became good practice to append the unit of time constats to the define name

```suggestion
#define NIMBLE_STATCONN_ADV_ITVL_MS         (90U)
```

Just a suggestion, not a must.